### ISSUE TITLE
dri alias should eliminate only empty images

### ISSUE THREAD

**Issue by ninrod** [tcnksm/docker-alias#10]
`dri` alias is very dangerous. most of the time user wants to eliminate only the unnamed images.

**Comment by neverfox** [comment]
Yeah, my dri is `docker images -qf dangling=true | xargs docker rmi;`
